keras/optimizers.py

```diff
@@ -48,19 +20,21 @@ class Optimizer(object):
 
     All Keras optimizers support the following keyword arguments:
 
-        clipnorm: float >= 0. Gradients will be clipped
-            when their L2 norm exceeds this value.
-        clipvalue: float >= 0. Gradients will be clipped
-            when their absolute value exceeds this value.
+        gradient_modifier: The gradient modifier to use
+            (see [gradient_modifiers](gradient_modifiers.md)).
     """
 
     def __init__(self, **kwargs):
-        allowed_kwargs = {'clipnorm', 'clipvalue'}
+        allowed_kwargs = {'gradient_modifier'}
```

Comment:
Also wondering if we want to maintain backwards compatibility here. Its easily done but at the cost of making the code unsightly.